lib/src/sqlite.rs

```diff
@@ -1,43 +1,41 @@
-use crate::types::core::{CapMessage, PackageId};
+use crate::types::core::PackageId;
 use rusqlite::types::{FromSql, FromSqlError, ToSql, ValueRef};
 use serde::{Deserialize, Serialize};
 use thiserror::Error;
 
 /// IPC Request format for the sqlite:distro:sys runtime module.
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct SqliteRequest {
     pub package_id: PackageId,
     pub db: String,
     pub action: SqliteAction,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum SqliteAction {
     Open,
     RemoveDb,
     Write {
         statement: String,
         tx_id: Option<u64>,
     },
```

Comment:
nit: same here as kv, should probably specify the type expected in the blob in a comment?